solidity/test/rewards/TestRewardUpgrades.js

```diff
@@ -168,6 +168,60 @@ describe("Rewards/Upgrades", () => {
             )
         })
 
+        it("allocates interval before finalize rewards upgrade", async () => {
```

Comment:
As far as I understand, what this test is checking, is whether the `finalizeRewardsUpgrade` works in case all previous intervals were allocated. It makes sense, though I think it's unclear from the test name. 

`finalizeRewardsUpgrade` do not allocate any interval in this case, because `await rewards.allocateRewards(1)` was called before `finalizeRewardsUpgrade`. 

The scenario with multiple intervals allocated is checked in the previous test (`"allocated all possible intervals"`).

What I think we can do here to improve the clarity is to, for example:
- rename `allocates all possible intervals` to `allocates all unallocated intervals when finalizing upgrade`,
- rename `allocates interval before finalize rewards upgrade` to `can be finalized with all previous intervals already allocated`. 

What do you think? It's possible I missed something.

Reply:
changed to `can be finalized with all previous intervals already allocated`